### PR TITLE
[Tools] Fix bug that translayer cannot detect bn

### PR DESCRIPTION
**Pull request by zhoonit**
For batchnormalization in tf 2.3 it is not detected in transLayer, so
added new type to detect batch normalization layer

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>

### CODE REVIEW

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by kparichay — APPROVED** [review]
LGTM
